spec/src/main/asciidoc/async.asciidoc

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright (c) 2018 Contributors to the Eclipse Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+[[restasync]]
+== MicroProfile Rest Client Asynchronous Support
+
+It is possible for Rest Client interface methods to be declared asynchronous.  This allows the thread invoking the interface method to proceed while the RESTful request occurs on another thread.
+
+=== Asynchronous Methods
+
+A method is considered to be asynchronous if either of the method's return type is `java.util.concurrent.CompletableFuture`.
```

Comment:
"either of the method's return type" sounds wrong, should be just "the method's return type".